src/chat/chat_panel.rs

```diff
@@ -147,39 +157,45 @@ live_design! {
         height: Fill,
         align: {x: 0.5, y: 1.0},
 
-        jump_to_bottom = <CircleView> {
+        jump_to_bottom = <RoundedButton> {
             width: 34,
             height: 34,
-            align: {x: 0.5, y: 0.5},
             margin: {bottom: 10},
 
-            cursor: Hand,
-
-            show_bg: true,
-
             draw_bg: {
-                radius: 14.0,
+                radius: 8.0,
                 color: #fff,
                 border_width: 1.0,
                 border_color: #EAECF0,
             }
-
-            <Icon> {
-                padding: 0,
-                // These margins are used to center the icon inside the circle
-                // Not sure why the icon is not centered by default
-                margin: { top: 6, right: 4 },
-                draw_icon: {
-                    svg_file: (ICON_JUMP_TO_BOTTOM),
-                    fn get_color(self) -> vec4 {
-                        return #1C1B1F;
-                    }
+            draw_icon: {
+                svg_file: (ICON_JUMP_TO_BOTTOM),
+                fn get_color(self) -> vec4 {
+                    return #1C1B1F;
                 }
-                icon_walk: {width: 12, height: 12}
+            }
+            icon_walk: {
+                margin: {top: 6, left: -4},
             }
         }
     }
 
+    PromptRoundedButton = <RoundedButton> {
```

Comment:
nitpick: Related to my previous comment, if the RoundedButton name changes this should change, but in this case I think the name could be as easy as `PromptButton`.